<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frm_Login</class>
 <widget class="QWidget" name="Frm_Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>系统登录</string>
  </property>
  <widget class="QPushButton" name="btn_Login">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>登  录</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_UserName">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>admin</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>173</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>密  码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_Pwd">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>123456</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Clear">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>210</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>清  空</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_bg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txt_UserName</tabstop>
  <tabstop>txt_Pwd</tabstop>
  <tabstop>btn_Login</tabstop>
  <tabstop>btn_Clear</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
